src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RSAPKCS1KeyExchangeFormatter.cs

```diff
@@ -26,6 +26,11 @@ public override string Parameters
             }
         }
 
+        public RandomNumberGenerator Rng {
+            get { return null; }
+            set { throw new PlatformNotSupportedException(); }
```

Comment:
Same question, but here it looks like the desktop implementation also Creates and stores one of these during key creation... should this as well?